ext/CMakeLists.txt

```diff
@@ -13,146 +13,162 @@ ENDIF (ENABLE_RPMDB OR ENABLE_RPMPKG)
-	repo_helix.c)
+    repo_helix.c)
     SET (libsolvext_HEADERS ${libsolvext_HEADERS}
-	repo_helix.h)
+    repo_helix.h)
 ENDIF (ENABLE_HELIXREPO)
 
 IF (ENABLE_MDKREPO)
     SET (libsolvext_SRCS ${libsolvext_SRCS}
-	repo_mdk.c)
+    repo_mdk.c)
     SET (libsolvext_HEADERS ${libsolvext_HEADERS}
-	repo_mdk.h)
+    repo_mdk.h)
 ENDIF (ENABLE_MDKREPO)
 
 IF (ENABLE_ARCHREPO)
     SET (libsolvext_SRCS ${libsolvext_SRCS}
-	repo_arch.c)
+    repo_arch.c)
     SET (libsolvext_HEADERS ${libsolvext_HEADERS}
-	repo_arch.h)
+    repo_arch.h)
 ENDIF (ENABLE_ARCHREPO)
 
 IF (ENABLE_CUDFREPO)
     SET (libsolvext_SRCS ${libsolvext_SRCS}
-	repo_cudf.c)
+    repo_cudf.c)
     SET (libsolvext_HEADERS ${libsolvext_HEADERS}
-	repo_cudf.h)
+    repo_cudf.h)
 ENDIF (ENABLE_CUDFREPO)
 
 IF (ENABLE_HAIKU)
     SET (libsolvext_SRCS ${libsolvext_SRCS}
-	repo_haiku.cpp)
+    repo_haiku.cpp)
     SET (libsolvext_HEADERS ${libsolvext_HEADERS}
-	repo_haiku.h)
+    repo_haiku.h)
 ENDIF (ENABLE_HAIKU)
 
 IF (ENABLE_APPDATA)
     SET (libsolvext_SRCS ${libsolvext_SRCS}
-	repo_appdata.c)
+    repo_appdata.c)
     SET (libsolvext_HEADERS ${libsolvext_HEADERS}
-	repo_appdata.h)
+    repo_appdata.h)
 ENDIF (ENABLE_APPDATA)
 
 IF (ENABLE_RPMMD OR ENABLE_SUSEREPO)
     SET (libsolvext_SRCS ${libsolvext_SRCS}
-	repodata_diskusage.c)
+    repodata_diskusage.c)
 ENDIF (ENABLE_RPMMD OR ENABLE_SUSEREPO)
 
 IF (ENABLE_RPMMD OR ENABLE_SUSEREPO OR ENABLE_APPDATA OR ENABLE_COMPS OR ENABLE_HELIXREPO OR ENABLE_MDKREPO)
     SET (libsolvext_SRCS ${libsolvext_SRCS}
-	solv_xmlparser.c)
+    solv_xmlparser.c)
 ENDIF (ENABLE_RPMMD OR ENABLE_SUSEREPO OR ENABLE_APPDATA OR ENABLE_COMPS OR ENABLE_HELIXREPO OR ENABLE_MDKREPO)
 
 IF (ENABLE_ZCHUNK_COMPRESSION)
     SET (libsolvext_SRCS ${libsolvext_SRCS}
-	solv_zchunk.c)
+    solv_zchunk.c)
 ENDIF (ENABLE_ZCHUNK_COMPRESSION)
 
 SET (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
 IF (HAVE_LINKER_VERSION_SCRIPT)
-SET (CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} ${LINK_FLAGS} -Wl,--version-script=${CMAKE_SOURCE_DIR}/ext/libsolvext.ver")
+    SET (CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} ${LINK_FLAGS} -Wl,--version-script=${CMAKE_SOURCE_DIR}/ext/libsolvext.ver")
 ENDIF (HAVE_LINKER_VERSION_SCRIPT)
 
 IF (DISABLE_SHARED)
-ADD_LIBRARY (libsolvext STATIC ${libsolvext_SRCS})
+    ADD_LIBRARY (libsolvext STATIC ${libsolvext_SRCS})
 ELSE (DISABLE_SHARED)
-ADD_LIBRARY (libsolvext SHARED ${libsolvext_SRCS})
-TARGET_LINK_LIBRARIES(libsolvext libsolv ${SYSTEM_LIBRARIES})
+    ADD_LIBRARY (libsolvext SHARED ${libsolvext_SRCS})
+    TARGET_LINK_LIBRARIES(libsolvext libsolv ${SYSTEM_LIBRARIES})
 ENDIF (DISABLE_SHARED)
 
-SET_TARGET_PROPERTIES(libsolvext PROPERTIES OUTPUT_NAME "solvext")
-SET_TARGET_PROPERTIES(libsolvext PROPERTIES SOVERSION ${LIBSOLVEXT_SOVERSION})
-SET_TARGET_PROPERTIES(libsolvext PROPERTIES INSTALL_NAME_DIR ${CMAKE_INSTALL_LIBDIR})
+SET_TARGET_PROPERTIES(libsolvext
+    PROPERTIES
+      OUTPUT_NAME "solvext"
+      MACOSX_RPATH ON
```

Comment:
Why do we want RPATH on? Usually we'd prefer to have it off...

Reply:
Let's try to remove it. I needed it for the conda-forge build to be able to properly link against it but this might be gone now that we're using the GNU Install Dirs.
I'm no expert when it comes to OS X and rpath handling. In the worst case I can probalby set a default value on the command line for the OS X build or patch the cmake.